src/js/modules/RoutingApp.js

```diff
@@ -1,49 +1,34 @@
-import Provider from 'react-redux/lib/components/Provider';
-import React, { Component } from 'react';
-import appHistory from 'tools/appHistory';
-import ArticleForm from './forms/components/ArticleForm';
-import SectionForm from './forms/components/SectionForm';
-import ArticlesTable from './articles/components/ArticlesTable'
-import UsersList from './users/components/UsersList'
-import ConnectedRouter from 'react-router-redux/ConnectedRouter';
-import store from '../store';
-import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
-import { Route } from 'react-router-dom'
-import MainApp from './main/components/MainApp'
+import Provider from "react-redux/lib/components/Provider";
+import React, { Component } from "react";
+import appHistory from "tools/appHistory";
+import ArticleForm from "./forms/components/ArticleForm";
+import SectionForm from "./forms/components/SectionForm";
+import ArticlesTable from "./articles/components/ArticlesTable";
+import UsersList from "./users/components/UsersList";
+import ConnectedRouter from "react-router-redux/ConnectedRouter";
+import store from "../store";
+import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
+import { Route } from "react-router-dom";
+import MainApp from "./main/components/MainApp";
 

```

Comment:
I hate redirection for imports. You're inherently hiding where the import is coming from then